sympy/printing/tests/test_octave.py

```diff
@@ -398,3 +398,12 @@ def test_MatrixElement_printing():
 
     F = C[0, 0].subs(C, A - B)
     assert mcode(F) == "(-B + A)(1, 1)"
+
+
+def test_zeta_printing():
+    # test cases for issue #14820
```

Comment:
can we just call the fcn `test_zeta_printing_14820():` and don't have the comment

Reply:
or maybe its `...printing_issue_14820`, look through other issues to see which is more standard.